functors_task_1/student/include/impl.h

```diff
@@ -1,13 +1,33 @@
 #pragma once
 #include "bar_serving.h"
 #include <functional>
+#include <vector>
 
 /**
  * @todo Implement functor-generator that will return next beer brand (cyclic)
  * @example HoeGaarden -> Corona -> Carlsberg -> Bud -> ZlataPraha -> Leffe -> HoeGaarden...
  */
 struct BeerOrganizer
 {
+    BeerBrand operator()()
+    {
+        static std::vector<BeerBrand> brands = {
```

Comment:
1. Please implement this task without containers usage
2. You don't need assignment
3. Why not const?
4. Why do you need vector, if the number of elements is known at compile time?
5. Why it is not auto-generated? What if there will be 50+ values?

Reply:
Done!